atlp-rwanda/e-commerce-mavericks-fn
# [Finishes #187788610] create buyers page on the admin-dashboard

## Purpose

Implement a Buyers Page on the admin dashboard that fetches and displays a list of buyers with their details.
![pull request](https://github.com/user-attachments/assets/9dc86ff0-50b3-430d-a93e-07bcf5f46068)


## Changes Made

- Created the `BuyersPage` component to fetch and display buyer data.
- Implemented search functionality to filter buyers by name.
- Added edit and delete icons for future implementation of respective functionalities.
- Utilized local storage to retrieve the authentication token for API requests.

## Testing Instructions

1. Ensure you have the authentication token stored in local storage.
2. Navigate to the Buyers Page on the admin dashboard.
3. Verify that the list of buyers is fetched and displayed correctly.
4. Use the search input to filter buyers by their first and last names.

## Related Issues

Lists all related issues

## Checklist

Please review the following checklist and make sure all tasks are complete before submitting:
